ydb/src/topic_create_delete_test.rs

```diff
@@ -0,0 +1,28 @@
+use tracing_test::traced_test;
+use crate::client_topic::create_topic_options::TopicOptionsBuilder;
+
+use crate::test_integration_helper::create_client;
+use crate::YdbResult;
+
+#[tokio::test]
+#[traced_test]
+#[ignore] // need YDB access
+async fn create_delete_topic_test() -> YdbResult<()>{
+    let client = create_client().await?;
+    let database_path = client.database();
+    let topic_name = "test_topic".to_string();
+    let topic_path = format!("{}/{}", database_path, topic_name);
+
+    let mut topic_client = client.topic_client();
+    let mut scheme_client = client.scheme_client();
+
+    topic_client.create_topic(topic_path.clone(), TopicOptionsBuilder::default().build()?).await?;
```

Comment:
it can have problem for run in not-empty (run on standalone database or persistent runned docker-container after previous run failed) and database will have "test_topic".

It will be better drop the topic before test (with ignore scheme error).